api/core/v2/tessen.go

```diff
@@ -5,14 +5,14 @@ import (
 )
 
 const (
-	// LowerBound is the minimum frequency that tessen will phone home
-	LowerBound = 1
+	// LowerBound is the minimum frequency that tessen will phone home, in seconds
+	LowerBound = 60
```

Comment:
I'm a bit confused by comment, why to use term `frequency` while it should be `period` (`f = 1 / t`)?

Reply:
I can update the verbiage for sure 😄

Reply:
Thank you! Learning Go is easy on Sensu, great work :)